conda modifies the build to put the dll in base/lib not base/bin

diff -r ab92c5997c51 lib_pypy/_sysconfigdata.py
--- a/lib_pypy/_sysconfigdata.py	Sun Feb 20 17:00:07 2022 +0200
+++ b/lib_pypy/_sysconfigdata.py	Sun Feb 20 17:27:41 2022 +0200
@@ -51,7 +51,7 @@
 else:
     build_time_vars['LDLIBRARY'] = 'libpypy3-c.so'
     build_time_vars['INCLUDEPY'] = os.path.join(mybase, 'include', 'pypy' + pydot)
-    build_time_vars['LIBDIR'] = os.path.join(mybase, 'bin')
+    build_time_vars['LIBDIR'] = os.path.join(mybase, 'lib')
     # try paths relative to sys.base_prefix first
     tzpaths = [
         os.path.join(mybase, 'share', 'zoneinfo'),

